godot-macros/src/class/data_models/field_export.rs

```diff
@@ -266,6 +267,11 @@ impl FieldExport {
         let min = parser.next_expr()?;
         let max = parser.next_expr()?;
 
+        let step = match parser.is_next_ident() {
+            Some(false) => parser.next_expr()?,
+            _ => TokenStream::new(),
+        };
+
```

Comment:
if you make this evaluate to `quote! { Some(#expr) }`/`quote! { None }` then we can avoid some duplication further down where this value is used.

Reply:
Done it, thanks for a hint :)